src/index.tsx

```diff
@@ -376,7 +376,9 @@ export function formAtom<Fields extends FormFields>(
     });
   }
 
-  return atom(formAtoms);
+  (formAtoms as Record<string, unknown>)._validateFields = validateFields;
```

Comment:
hmm, this is why i think it's better to add it to the object at L359. casts are fake types and the point of the explicit return value is to enforce correctness. i would add it to the object and then be ok with the debug label thing not being perfect because that's development-only anyway.